### PR TITLE
Added load spake credentials from flash

### PR DESCRIPTION
**Pull request by Skyler84**
spake credentials are loaded from flash at program startup. (requires waiting for next sdk release)

### CODE REVIEW

File: EXAMPLE/readme.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# The Application
+# The Applications
```
Comment (elie-elkhoury) [inline EXAMPLE/readme.md]:
```suggestion
# The Application
```

File: EXAMPLE/knx_iot_wakeful_main.c [hunk 2]
```diff
@@ -308,6 +308,39 @@ int main(void)
...
+	}
+	
+	uint8_t salt[32], rand[32];
+	uint32_t it;
+	mbedtls_mpi w0;
+	mbedtls_ecp_point L;
+	mbedtls_mpi_init(&w0);
+	mbedtls_ecp_point_init(&L);
+	error = knx_get_stored_spake(salt, rand, &it, &w0, &L);
+	if (error)
```
Comment (elie-elkhoury) [inline EXAMPLE/knx_iot_wakeful_main.c:332]: I think you probably also want to free `w0` and `L` in the error case. (Likewise for sleepy_main.c)

**Review by elie-elkhoury — CHANGES_REQUESTED** [review]
Nice one! Looks good to me, I just have a couple very minor comments. 

Otherwise, I'm happy to approve once a new release of cascoda-sdk is made, and the PR passes CI.

File: EXAMPLE/knx_iot_wakeful_main.c (hunk 2 above)
Comment (Skyler84) [reply EXAMPLE/knx_iot_wakeful_main.c:332]: This is already handled in the SDK - any error cases dealloc anything used or returned. I can still add it in though if you'd prefer

Comment (elie-elkhoury) [reply EXAMPLE/knx_iot_wakeful_main.c:332]: Ah I see, that's good enough then, no need to add it here.

**Review by elie-elkhoury — APPROVED** [review]

**Review by tiniuclx — APPROVED** [review]

**Review by WAvdBeek — APPROVED** [review]

File: EXAMPLE/knx_iot_wakeful_main.c [hunk 3]
```diff
@@ -308,6 +308,39 @@ int main(void)
...
+	}
+	else
+	{
+		oc_spake_set_parameters(rand, salt, it, w0, L);
+		mbedtls_mpi_free(&w0);
+		mbedtls_ecp_point_free(&L);
+	}
 
 	/* set the application callbacks */
 	oc_set_hostname_cb(hostname_cb, NULL);
```
Comment (WAvdBeek) [inline EXAMPLE/knx_iot_wakeful_main.c:346]: note: the build is not working

**Review by WAvdBeek — CHANGES_REQUESTED** [review]

File: EXAMPLE/knx_iot_wakeful_main.c (hunk 3 above)
Comment (tiniuclx) [reply EXAMPLE/knx_iot_wakeful_main.c:346]: We know! Gotta wait for a public SDK release, which is dependant on Skyler's current task

**Pull request closed without merging**